aspnetcore/tutorials/web-api-vsc.md

```diff
@@ -79,6 +79,8 @@ Add a `TodoItem` class with the following code:
 
 [!code-csharp[Main](first-web-api/sample/TodoApi/Models/TodoItem.cs)]
 
+The database generates the `Id` when a `TodoItem` is created.
```

Comment:
Seems like it would be more useful to have this under the Create code snippet in "Implement the other CRUD operations / Create" section, to explain how item.Id ends up with a value for CreatedAtRoute.